### PR TITLE
Issue #67 fix: Allows entering text in calendar field 

### PR DESCRIPTION
**Pull request by vaibhavsTekdi**
**Steps to reproduce**

- Go to the admin panel.
- Add new calendar field under particular UCM type.
- Go to the user panel, you can see the date field on that particular UCM type

Here the user can enter text/characters in a calendar field.

### CODE REVIEW

**Comment by vaibhavsTekdi** [comment]
We have already written a validation class for the fields like calendar, file, number etc. Whenever admin adds a new field/update the existing one, it will automatically set the validation class(if available) value in the "Validation Class" config under "Extra Fields Attributes" section.

File: administrator/views/field/tmpl/edit.php [hunk 1]
```diff
@@ -283,7 +287,22 @@ function showOptions()
 							<div class="control-group">
 								<div class="control-label"><?php echo $this->form->getLabel('validation_class'); ?></div>
 								<div class="controls">
-									<?php echo $this->form->getInput('validation_class'); ?>
+									<?php
+									switch ($fieldType) {
```
Comment (ankush-maherwal) [inline administrator/views/field/tmpl/edit.php]: @vaibhavsTekdi 
cant we do this from XML?

Comment (ankush-maherwal) [inline administrator/views/field/tmpl/edit.php]: @vaibhavsTekdi 
can we avoid using switch case?

File: administrator/views/field/tmpl/edit.php [hunk 2]
```diff
@@ -283,7 +287,22 @@ function showOptions()
...
+										case "calendar":
+											$validationClassValue = "calendar-textfield-class";
+											break;
+										case "file":
+											$validationClassValue = "validate-filetype";
+											break;
+										case "number":
+											$validationClassValue = "validate-check_number_field";
```
Comment (ankush-maherwal) [inline administrator/views/field/tmpl/edit.php]: @vaibhavsTekdi 
class is not used anywhere?

why are we adding this?

Comment (vaibhavsTekdi) [reply administrator/views/field/tmpl/edit.php]: @ankush-maherwal while adding the new field, these are the fields where jQuery validation classes are needed. This function automatically adds the validation class on field type selection. 

The reason behind doing this is, after the creation of field most of the time admin doesn't know validation class name if any or he/she can forget to add the validation class. In this scenario, the end user can add any value against that particular field.

Comment (ankush-maherwal) [reply administrator/views/field/tmpl/edit.php]: @vaibhavsTekdi 
is any JS or CSS added against class "validate-check_number_field" in TJ-Fields or else admin is supposed to add JS for validation?

**Comment by ankush-maherwal** [comment]
closing this MR as the code is obsolete

**Pull request closed without merging**